chromecast/src/main/java/com/aol/mobile/sdk/chromecast/OneCastManager.java

```diff
@@ -26,18 +27,24 @@
 import com.google.android.gms.cast.framework.SessionManagerListener;
 
 public class OneCastManager {
-
+    @NonNull
+    private Context context;
+    @Nullable
     private SessionManagerListener sessionManagerListener;
 
-    public static View getCastButton(@NonNull Context context) {
+    public OneCastManager(@NonNull Context context) {
+        this.context = context;
+    }
+
+    public View getCastButton() {
```

Comment:
`construct` will be better then `get`, because each subsequent call will return new instance of `MediaRouteButton `